ui/src/pandas/collection/TitleController.java

```diff
@@ -137,7 +141,26 @@ public RedirectView pandas3(@PathVariable("id") long id) {
     @GetMapping("/titles")
     public String search(@RequestParam MultiValueMap<String, String> params,
                          @PageableDefault(20) Pageable pageable,
-                         Model model) {
+                         Model model, HttpServletRequest request) {
+    	
+    	// if called without parameters, redirect to the last saved sticky parameters
+    	if (params.isEmpty())
+    	{
```

Comment:
I guess we should probably standardise on [K & R style blocks](https://google.github.io/styleguide/javaguide.html#s4.1.2-blocks-k-r-style) for consistency with the rest of the project and other NLA code.

```suggestion
    	if (params.isEmpty()) {
```

Reply:
Indentation should be fixed now. I've set Eclipse to to spaces only, and reformatted the new code.